src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/ConstantPool.java

```diff
@@ -131,7 +131,24 @@ default JavaMethod lookupMethod(int cpi, int opcode) {
 
     /**
      * The details for invoking a bootstrap method associated with a {@code CONSTANT_Dynamic_info}
-     * or {@code CONSTANT_InvokeDynamic_info} pool entry .
+     * or {@code CONSTANT_InvokeDynamic_info} pool entry.
+     *
+     * The procedure to obtain and use a {@link BootstrapMethodInvocation} is the following:
+     *
+     * <pre>
+     * bsmInvocation = lookupBootstrapMethodInvocation(index, opcode);
+     * staticArguments = bsmInvocation.getStaticArguments();
+     * if staticArguments are PrimitiveConstant {
+     *     argCount = staticArguments.get(0).asInt();
+     *     cpi = staticArguments.get(1).asInt();
+     *     for (int i = 0; i < argCount; ++i) {
+     *         arguments[i] = lookupConstant(cpi, i);
```

Comment:
What `lookupConstant` method is this?
Where is the usage of the new `bootstrapArgumentIndexAt` method?

Reply:
I made a mistake sorry, it is now correct.
The idea is to use `bootstrapArgumentIndexAt` to obtain the constant pool index of the static argument and use it to call `ConstantPool.lookupConstant(int cpi, boolean resolve)` to get the constant.